lib/jekyll/commands/clean.rb

```diff
@@ -17,13 +18,16 @@ def init_with_program(prog)
 
         def process(options)
           options = configuration_from_options(options)
-          destination = options['destination']
-          metadata_file = File.join(options['source'], '.jekyll-metadata')
-          sass_cache = File.join(options['source'], '.sass-cache')
+          destination = options["destination"]
+          metadata_file = File.join(options["source"], ".jekyll-metadata")
+          sass_cache = File.join(options["source"], ".sass-cache")
 
+          # Disabling rubocop due to https://github.com/bbatsov/rubocop/issues/1451
+          # rubocop:disable Style/HashSyntax
           remove(destination, checker_func: :directory?)
           remove(metadata_file, checker_func: :file?)
           remove(sass_cache, checker_func: :directory?)
```

Comment:
Huh. Learned something new. I figured the keyword arg syntax would've made this acceptable style, but I can see your point.